pravega-docker/up.sh

```diff
@@ -14,9 +14,14 @@ set -ex
 ROOT_DIR=$(readlink -f $(dirname $0))
 cd ${ROOT_DIR}
 export HOST_IP=${HOST_IP:-192.168.120.128}
-export PRAVEGA_LTS_PATH=${PRAVEGA_LTS_PATH:-/tmp/pravega-lts}
-docker-compose down
-sudo rm -rf ${PRAVEGA_LTS_PATH}
+export PRAVEGA_LTS_PATH=${PRAVEGA_LTS_PATH-/tmp/pravega-lts}
```

Comment:
the `-` alone makes this statement mean "if PRAVEGA_LTS_PATH isn't set, set it to /tmp/pravega-lts`.

Reply:
Can you explain the difference between `:-` and `-`?